lib/matplotlib/image.py

```diff
@@ -688,6 +688,34 @@ def write_png(self, fname):
                           bytes=True, norm=True)
         PIL.Image.fromarray(im).save(fname, format="png")
 
+    @staticmethod
+    def _prepare_array(A):
+        # Common checks and typecasts on A for various Image subclasses.
```

Comment:
"prepare_array" is a bit vague.

```suggestion
    def _normalize_image_array(A):
        """
        Transform image-like input into a normalized array represenation,
        that is used as data format in Image subclasses.
        """
```

Reply:
Sure.

Reply:
Did you intentionally not take the docstring?

Reply:
oops, missed it.  reworded and included now.